pyfolio/deprecate.py

```diff
@@ -0,0 +1,45 @@
+"""Utilities for marking deprecated functions."""
+# Copyright 2016 Quantopian, Inc.
```

Comment:
@richafrank Does this all mean each file in qrisk should have a copyright notice?